### PR TITLE
fix: change protobuf v2 for lib as-proto support

### PR DESCRIPTION
**Pull request by gene-zhan**
Change protobuf v2 for support  as-proto

### CODE REVIEW

File: proto/message/aspect/v2/host_api.proto
Comment (dumbeng) [inline proto/message/aspect/v2/host_api.proto:1]: format this file before commit

File: proto/message/aspect/v2/joinpoint_input.proto
Comment (dumbeng) [inline proto/message/aspect/v2/joinpoint_input.proto:1]: format this file before commit

**Review by dumbeng — CHANGES_REQUESTED** [review]

**Review by dumbeng — APPROVED** [review]
LGTM

**Review by luketheart — APPROVED** [review]
LGTM.

**Pull request merged**